eclipse-projects/cli-test/.project

```diff
@@ -36,5 +36,10 @@
 			<type>2</type>
 			<locationURI>DAFFODIL_ROOT/daffodil-cli/src/test</locationURI>
 		</link>
+		<link>
+			<name>src/it</name>
+			<type>2</type>
+			<locationURI>DAFFODIL_ROOT/daffodil-cli/src/it</locationURI>
+		</link>
```

Comment:
Mike, does this and the .classpath modification look correct to you? I don't have an eclipse setup to test. Basically just copied things that had src/test and changed to src/it

Reply:
Looks right. When you've committed this ping me and I'll verify this right away. but I think this is right and it's just going to work.  One does not usually run these tests from eclipse, so really this is just making sure eclipse compiles these along with everything else when changes are made.